AmplifyPlugins/Core/AWSPluginsCore/Model/Support/GraphQLDocumentnputValue.swift

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+/// A container to hold either an object or a value, useful for storing document inputs and allowing manipulation at
+/// the first level of the object
+public enum GraphQLDocumentInputValue {
+    case value(Any)
+    case object([String: Any?])
```

Comment:
Is it meaningful to distinguish between `value` and `Any` when `Any` could itself be a `[String: Any?]`? Seems like we should specify we're expecting a scalar value, or a String.